packages/evo/src/__tests__/llm.spec.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  OpenAI,
+  Chat,
+  ConsoleLogger,
+  Logger,
+  InMemoryWorkspace,
+  Env,
+  ChatRole,
+  LlmResponse
+} from "@evo-ninja/agent-utils";
+import dotenv from "dotenv";
+import cl100k_base from "gpt-tokenizer/cjs/encoding/cl100k_base";
+import path from "path";
+import { agentFunctions } from "../agent-functions";
+
+dotenv.config({
+  path: path.join(__dirname, "../../../../.env")
+});
+
+describe('AI Agent Test Suite', () => {
+  const oneMinute = 300 * 1000;
+
+  afterAll(() => {
+  
+  });
+  test(`Execute`, async() => {
+    const msgs = {
```

Comment:
Sure, but these aren't really tests, it's more of a tool to test out features while developing.
These can never really pass cause with non 0 temp the output is non deterministic, and if we put the temp to 0 it can still change if the prompts change slightly. We don't gain much by testing things like this ATM.

Reply:
Ah okay, understood